util/fibers/detail/scheduler.cc

```diff
@@ -416,15 +417,17 @@ void DispatcherImpl::DefaultDispatch(Scheduler* sched) {
       sched->ProcessSleep();
     }
 
+    uint64_t now = absl::GetCurrentTimeNanos();
+
     if (sched->HasReady()) {
       FiberInterface* fi = sched->PopReady();
       DCHECK(!fi->list_hook.is_linked());
       DCHECK(!fi->sleep_hook.is_linked());
-      sched->AddReady(this);
+      sched->AddReady(now, this);
 
       DVLOG(2) << "Switching to " << fi->name();
       // qsbr_worker_fiber_online();
-      fi->SwitchTo();
+      fi->SwitchTo(now);
```

Comment:
This causes the CHECK failure in the test, I guess the CHECK should be GE rather than GT?